### DISCUSSION TITLE
Conversion of values stored as 0 and 1 to Bool?

### DISCUSSION THREAD

**Discussion by mirozahorak** [flutterdata/flutter_data#202] [General]
I am trying to find a documentation about this, can someone point me the right direction?
I have a bool value stored in the DB and need it converted to bool true/false values by the generated classes.

I am sure this popped up somewhere before, but there is not much content about flutter_data on internet. The name of library also makes search very difficult.

**Comment by frank06** [comment]
Simply use `json_serializable` converters: https://github.com/google/json_serializable.dart/blob/master/example/lib/json_converter_example.dart
